nat-lab/tests/utils/vm/windows_vm_util.py

```diff
@@ -44,9 +44,12 @@ async def new_connection(
     ) as ssh_connection:
         connection = SshConnection(ssh_connection, TargetOS.Windows)
 
+        keys = await _get_network_interface_tunnel_keys(connection)
+        for key in keys:
+            await connection.create_process(["reg", "delete", key, "/F"]).execute()
```

Comment:
nit: Don't think this is the best place to put this here. Such things shouldn't be done during test execution. Maybe not during this PR, but it should be moved to test setup/teardown :thinking:

Reply:
I will create a jira ticket to move this to test setup.